data/datacatalogue/stagingShared/molgenis.csv

```diff
@@ -1,14 +1,16 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description
-Version,,,,,,,,,,,,2.8
+Version,,,,,,,,,,,,2.9
 Contacts,,,,,,,,,,,,
 Contacts,,firstName,,1,TRUE,,,,,,,First name of the contact person
 Contacts,,surname,,1,TRUE,,,,,,,Last name of the contact person
+Contacts,,consent,bool,,TRUE,,,,,,,"This person gives permission to display their filled out contact details in the catalogue, which is openly accessible"
```

Comment:
maybe make this 'consent display'

Reply:
I will revert these changes, because we no not want these options in the staging areas of the data catalogue, only in the UMCG staging areas.